Makefile

```diff
@@ -176,8 +176,8 @@ coverage-build: all
 		--single-branch git://github.com/gcovr/gcovr.git; fi
```

Comment:
We possibly can put a commit hash of the current latest `gcovr` at here for this change, then we can keep it as is a bit more easily.